eclipse-bluechi/workspace/workloads/trailer_connected_provider.kube

```diff
@@ -9,5 +9,8 @@
 Yaml=trailer_connected_provider.yml
 
 # Commented to disable the service to automatically start
-# [Install]
-# WantedBy=default.target
+[Install]
+WantedBy=default.target
+
+[Unit]
+Wants=agemo.kube
```

Comment:
I think that the dependencies in the sample workloads should be ibeji

All of them interact with Ibeji and Chariott